app/models/condition.rb

```diff
@@ -87,7 +87,10 @@ def self.evaluate(cond, rec, inputs={}, attr=:expression)
   def self.do_eval(expr)
     # Remove this [DEPRECATION] in the next version
     if expr =~ /^__start_ruby__\s?__start_context__\s?(.*)\s?__type__\s?(.*)\s?__end_context__\s?__start_script__\s?(.*)\s?__end_script__\s?__end_ruby__$/im
-      raise "Ruby scripts in expressions are no longer supported. Please use the regular expression feature of conditions instead."
+      context, col_type, script = [$1, $2, $3]
+      context = MiqExpression.quote(context, col_type)
+      result = SafeNamespace.eval_script(script, context)
```

Comment:
> We thought that the feature was not being used anymore and that's why Bill had removed it. But, then when I was working on the BZ referenced above, I realized that the feature was still necessary.

I thought we removed it because we wanted to kill it, and having discovered it was still being used for a desired feature, we were going to implement a "better" solution. :confused: